test/reductions/diten/reduction_name_collision.chpl

```diff
@@ -13,12 +13,12 @@ delete s;
 delete r;
 
 class R {
-  var count: sync int = 0;
+  var count: sync int = 0;  // TODO: initializer
```

Comment:
@bradcray - here too

Reply:
Yes to removing all three of these TODO notes, thanks.  I believe that each of these was indicating a case where my branch wasn't working, and where I could choose to work around it by introducing an explicit initializer, but that I didn't want to take that step since I thought it was a case where the user shouldn't need to change anything.  At some point, I kept re-diagnosing the same failure cases so started making notes like this to myself to capture the diagnosis, I believe.